README.md

```diff
@@ -39,6 +39,7 @@ The following tasks are no longer being developed and will be deprecated in a fu
 |-----------|-------------|
 | st0236_set_cache_paths_to_default | Use [puppet conf](https://forge.puppet.com/modules/puppetlabs/puppet_conf/readme) |
 | st0267_clear_file_sync_locks | See [knowledge article](https://support.puppet.com/hc/en-us/articles/360003883933) for manual steps |
+| st0305_support_script_and_upload | MFT and P1 are the preferred methods for upload, see [MFT Documentation](https://portal.perforce.com/s/article/360009970114) for steps| 
```

Comment:
can this be changed to something like see the following for upload methods?  SFTP and MFT are the preferred options. and sftp has the same function built into to pe that this script used to perform